pages/stack/interop/_meta.json

```
@@ -2,5 +2,6 @@
     "explainer": "Interop explainer",
     "cross-chain-message": "Anatomy of cross-chain message",
     "supersim": "Supersim Multichain Development Environment",
-    "superchain-erc20": "SuperchainERC20"
+    "superchain-erc20": "SuperchainERC20 token standard",
```

just "SuperchainERC20" (no token standard)
```suggestion
    "superchain-erc20": "SuperchainERC20",
```

---

we're not using 'token standard' per product team